tests/test_forward_sampling.py

```diff
@@ -189,19 +192,23 @@ def test_sample_posterior_predictive(posterior_predictive_fixture):
         observed_kwarg = None
     else:
         observed_kwarg = observed
-    ppc = pm.sample_posterior_predictive(model(), trace, observed=observed_kwarg)
+    ppc = pm.sample_posterior_predictive(
+        model(), trace, observed=observed_kwarg
+    ).posterior_predictive
     assert set(sorted(list(ppc))) == set(observed)
     assert np.all(
-        [v.shape == (num_samples, num_chains) + observed[k].shape for k, v in ppc.items()]
+        [v.shape == (num_chains, num_samples) + observed[k].shape for k, v in ppc.items()]
     )
 
 
 def test_sample_ppc_var_names(model_fixture):
     model, observed = model_fixture
-    trace = {
-        "model/sd": tf.convert_to_tensor(np.array(1.0, dtype="float32")),
-        "model/y": tf.convert_to_tensor(observed),
-    }
+    trace = pm.inference.utils.trace_to_arviz(
+        {
+            "model/sd": tf.ones((10, 1), dtype="float32"),
```

Comment:
Why is the singleton axis needed here?

Reply:
because we have one chain

Reply:
Do we always need the chain axis in an inference data object?

Reply:
yeah, for now we do - we can avoid this if we do the translation inside pymc4